csi/controller_server.go

```diff
@@ -1069,19 +1070,28 @@ func (cs *ControllerServer) cleanupSnapshot(sourceVolumeName, id string) error {
 	return nil
 }
 
-func (cs *ControllerServer) cleanupBackupVolume(sourceVolumeName, id string) error {
+func (cs *ControllerServer) cleanupBackup(sourceVolumeName, id string) error {
 	backupVolumeName, backupName := sourceVolumeName, id
-	backupVolume, err := cs.getBackupVolume(backupVolumeName)
+	backupVolumes, err := cs.getBackupVolumes(backupVolumeName)
 	if err != nil {
 		return err
 	}
-	if backupVolume != nil && backupVolume.Name != "" {
-		if _, err = cs.apiClient.BackupVolume.ActionBackupDelete(backupVolume, &longhornclient.BackupInput{
-			Name: backupName,
-		}); err != nil {
-			return err
+	for _, bv := range backupVolumes {
```

Comment:
Can we leave a log that the backup `id` is not deleted if the list `backupVolumes` is empty?

Reply:
If the backvolume is empty, Longhorn would have already cleanup all the backup. The comment might be misleading?

Reply:
>If the backvolume is empty, Longhorn would have already cleanup all the backup

Agree.